src/gui/export/kml.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 30 16:32:42 2017
+
+@author: PCHP
+"""
+
+"""
```

Comment:
Can we keep the same header as in other files? In some other words, ca nwe remove from L2 to L8